browser/app/js/browser/ChangePasswordModal.js

```diff
@@ -42,7 +42,7 @@ export class ChangePasswordModal extends React.Component {
     const { serverInfo } = this.props
 
     // Check environment variables first.
-    if (serverInfo.info.isEnvCreds) {
+    if (serverInfo.info.isEnvCreds || serverInfo.info.isWorm) {
```

Comment:
shouldn't this be !serverInfo.info.isWorm - in admin handlers, UpdateCredentialsHandler writes an error response if creds are being changed.

Reply:
nvm - didnt notice that this code path is just setting keys to xxx

Reply:
This only tells react to simply enable readOnly to the keys.